platform/macos/os_macos.mm

```diff
@@ -757,6 +757,19 @@
 	return PREFERRED_TEXTURE_FORMAT_S3TC_BPTC;
 }
 
+#ifdef DEBUG_ENABLED
+String OS_MacOS::get_debug_descriptor(const StackInfo &p_info) {
+	String pipe;
+	Error error = execute("atos", { "-o", p_info.file, "-l", String::num_uint64(reinterpret_cast<uint64_t>(p_info.load_address), 16), String::num_uint64(reinterpret_cast<uint64_t>(p_info.symbol_address), 16) }, &pipe);
```

Comment:
`atos` call probably should have `-arch arm64` or `-arch x86_64` arg added based on the running executable architecture, or in some conditions (like running under Rosetta) it might try reading wrong architecture from the fat binary.